lib/internal/Magento/Framework/Amqp/Connection/Factory.php

```diff
@@ -31,13 +31,11 @@ public function create(FactoryOptions $options): AbstractConnection
             'port' => $options->getPort(),
             'user' => $options->getUsername(),
             'password' => $options->getPassword(),
-            'vhost' => $options->getVirtualHost() !== null ? $options->getVirtualHost() : '/',
+            'vhost' => $options->getVirtualHost() ?? '/',
```

Comment:
We're now calling `getVirtualHost` just once, this is why one of the tests did break.